src/ensembl/src/content/app/entity-viewer/components/entity-viewer-topbar/scale-switcher/ScaleSwitcher.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/styles/common';
+
+.container {
+  width: 100%;
```

Comment:
this class is applied to a div; div has display: block by default, which has width: 100% by default.

Reply:
Yep agreed. I just realised it was partly finished code cleanup that i have pushed. But keep the comment coming i am pretty much there with the cleanup.